platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/platform/PlatformTestingToolState.java

```diff
@@ -1388,11 +1388,7 @@ private static boolean validateSignatures(final Transaction trans) {
     public void rebuildExpectedMapFromState(final Instant consensusTimestamp, final boolean isRestart) {
         // rebuild ExpectedMap
         logger.info(LOGM_DEMO_INFO, "Start Rebuilding ExpectedMap");
-        long timestampMilliseconds = 0;
-        if (consensusTimestamp != null) {
-            timestampMilliseconds = consensusTimestamp.toEpochMilli();
-        }
-        this.expectedFCMFamily.rebuildExpectedMap(getStateMap(), isRestart, timestampMilliseconds);
+        this.expectedFCMFamily.rebuildExpectedMap(getStateMap(), isRestart, 0);
```

Comment:
could you write a brief comment about why we always pass 0 in instead of `consensusTimestamp`?